config/projects.yml

```diff
@@ -387,9 +389,9 @@ translation-bundle:
         sonata_core: ['3']
         sonata_admin: ['3']
     2.x:
-      php: ['5.4', '5.5', '5.6', '7.0', '7.1']
+      php: ['5.6', '7.0', '7.1']
       versions:
-        symfony: ['2.3', '2.7', '2.8', '3.2', '3.3']
+        symfony: ['2.8', '3.1', '3.2']
```

Comment:
Symfony 3.1 is EOL

Reply:
Woops that one was unintended

Reply:
Had an old version, and made my changes before rebasing.